qiskit/qpy/__init__.py

```diff
@@ -17,6 +17,12 @@
 
 .. currentmodule:: qiskit.qpy
 
+QPY is a binary serialization format for :class:`~.QuantumCircuit` and
+:class:`~.ScheduleBlock` objects that is designed to be cross-platform,
+Python version agnostic, and backwards compatible moving forward. QPY should
+be used if you need a lossless mechanism to save or copy between systems a
```

Comment:
For end-users, it would be better to describe how this "lossless" use case differs from qasm3 exporter/importer?